export const signup = async (req, res) => {
    try {


        
    } catch (error) {
        console.log(error)
        res.status(400).json({
            error: error
        })
    }
}

export const signin = async (req, res) => {
    try {


        
    } catch (error) {
        console.log(error)
        res.status(400).json({
            error: error
        })
    }
}

export const signout = async (req, res) => {
    try {


        
    } catch (error) {
        console.log(error)
        res.status(400).json({
            error: error
        })
    }
}